test_xdb/models/under_test/clone_schema_test.sql

```diff
@@ -259,4 +259,7 @@ WITH all_objects_metadata AS (
 )
 
 SELECT *
-FROM final_stats
\ No newline at end of file
+FROM final_stats
+{%if target.type == 'snowflake' -%}
+WHERE object_type NOT IN ('view', 'function')
+{% endif %}
```

Comment:
Do views and functions now show up in the snowflake version?

Reply:
Yes, because `CLONE SCHEMA` statement also copies these objects, but such single ones as `CLONE FUNCTION` or `CLONE VIEW`, witch supposed to be generated in the loop after this current update, don't implemented and don't work. Previously I've implemented cloning for these kind of objects for both of DBs, but now because of the mentioned peculiarity decided to change behaviour for Snowflake, but keep it for Postgres. This update reflects this decision on the tests level.